unikernel.ml

```diff
@@ -930,31 +930,31 @@ struct
         | None ->
             let error =
               {
-                Utils.Status.code = 401;
+                Utils.Status.code = 404;
                 title = "Unauthenticated";
                 success = false;
                 data = `String "Auth cookie not found";
               }
             in
             reply reqd ~content_type:"text/html"
-              (Guest_layout.guest_layout ~page_title:"401 | Mollymawk"
+              (Guest_layout.guest_layout ~page_title:"404 | Mollymawk"
                  ~content:(Error_page.error_layout error)
                  ~icon:"/images/robur.png" ())
-              `Unauthorized)
+              `Not_found)
     | Error (`Msg err) ->
```

Comment:
Here the `error.title` is still "Unauthenticated" even though the error code was changes to not found / 404. There are other places with the same mistake.